jsdomClient/src/main/scala/HttpRpcTransport.scala

```diff
@@ -19,7 +19,7 @@ object HttpRpcTransport {
   def apply[F[_]: Async](config: F[HttpRequestConfig]): RequestTransport[String, F] = new RequestTransport[String, F] {
     override def apply(request: Request[String]): F[String] = for {
       config <- config
-      url = s"${config.baseUri}/${request.method.traitName}/${request.method.methodName}"
+      url = s"${config.baseUri}${request.method.traitName}/${request.method.methodName}"
```

Comment:
Good catch!

How about we do config.baseUri.trimSuffix("/") to make sure there is non? So that it works when providing a baseUri without a trailing slash.

Reply:
sounds like too many heuristics for me...